cmd/minishift/cmd/addon/addon_install.go

```diff
@@ -51,7 +51,7 @@ var addonsInstallCmd = &cobra.Command{
 func init() {
 	addonsInstallCmd.Flags().BoolVar(&force, forceFlag, false, "Forces the installation of the add-on even if the add-on was previously installed.")
 	addonsInstallCmd.Flags().BoolVar(&enable, enableFlag, false, "If true, installs and enables the specified add-on with the default priority.")
-	addonsInstallCmd.Flags().BoolVar(&defaults, defaultsFlag, false, "If true, installs all Minishift default add-ons.")
+	addonsInstallCmd.Flags().BoolVar(&defaults, defaultsFlag, false, "If true, installs all Minishift default add-ons.(default addons : anyuid, admin-user, xpaas, registry-route)")
```

Comment:
S/(default addons : /(default : /
Just `default:` is fine.

Reply:
Also no need to mention Minishift explicitly as it is understood. Suggestion `If true, installs the default add-ons.`

Reply:
Can these be specified at the top of the file as a const? These have special meaning and might have to change.

Reply:
@anjannath ,

Use `DefaultAssets` from https://github.com/minishift/minishift/blob/master/cmd/minishift/cmd/util/util.go#L34